README.md

```diff
@@ -12,23 +12,22 @@ cd bluesky
 ```
 
 ```bash
-conda create -n your_env_name python=3.10
+conda create -n your_env_name "python<13" "pyqt=5"
```

Comment:
Probably better to lock a version for now. I think could be 13 but need to see if it will resolve all other long list of bluesky dependencies. 
I would also add `export YOUR_ENV_NAME`=your_env_name to make the lines more consistent.